src/libs/DateUtils.js

```diff
@@ -291,6 +291,105 @@ function getDateStringFromISOTimestamp(isoTimestamp) {
     return dateString;
 }
 
+/**
+ * @returns {string} example: 2023-05-16
```

Comment:
Little comment tweak: the examples on this and a few others don't match the `format` used.

Reply:
It slipped from my eye, thanks!